src/components/Modal/ModalPortal.js

```diff
@@ -11,8 +11,11 @@ class ModalPortal extends React.Component {
     super(props);
 
     this.modalContainer = document.createElement('div');
-    if (this.props.className) {
-      this.modalContainer.className = this.props.className;
+    if (props.className) {
+      this.modalContainer.className = props.className;
+    }
+    if (props.zIndex != null) {
```

Comment:
shouldn't you check `null` or `undefined`?

Reply:
if I used triple equality (`===`/`!==`) then I would have to check for both `null` and `undefined`, but since `null == undefined` is `true` then only null-check is required

I can change that to strict equality if you prefer

Reply:
<img width="402" alt="x rozne null" src="https://user-images.githubusercontent.com/24607370/65140246-6d520580-da0e-11e9-8874-e7ab53995d62.png">
yup, please use `!==`

Reply:
wait, `x != null` is `true` which is fine (the z-index will be assigned '0')

Reply:
omg ;D sorry, you are right